spec/hooks_spec.rb

```diff
@@ -329,28 +329,6 @@
     end
   end
 
-  context 'when is from Google Divre Planilhas' do
-    let(:lead) do
-      lead = OpenStruct.new
-      lead.source = source
-      lead.message = 'Ribeirão Preto'
-      lead.description = nil
-
-      lead
-    end
-
-    let(:source) do
-      source = OpenStruct.new
-      source.name = 'Google Drive Planilha - Planilha - Simmons'
-
-      source
-    end
-
-    it 'returns source name' do
-      expect(switch_source).to eq('Google Drive Planilha - Planilha - Simmons - Ribeirão Preto')
-    end
-  end
-
```

Comment:
Remove this test because it was not working in f1sales and the customer did not report it